package.json

```diff
@@ -66,45 +63,31 @@
     "babel-preset-react": "^6.23.0",
     "babel-register": "^6.24.0",
     "chai": "^3.5.0",
-    "chai-enzyme": "^0.6.1",
-    "css-loader": "^0.27.3",
-    "cssnano": "^3.10.0",
-    "enzyme": "^2.8.0",
+    "enzyme": "^2.8.2",
     "eslint": "^3.17.1",
     "eslint-config-airbnb": "^14.1.0",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0",
-    "extract-text-webpack-plugin": "^2.0.0-rc.3",
-    "git-directory-deploy": "^1.5.0",
+    "gh-pages": "^0.12.0",
+    "in-publish": "^2.0.0",
     "jsdom": "^9.11.0",
-    "jss": "^6.5.0",
-    "jss-preset-default": "^1.3.1",
-    "jss-theme-reactor": "^0.10.0",
-    "lodash-webpack-plugin": "^0.11.2",
+    "jss": "^7.0.3",
+    "jss-preset-default": "^2.0.0",
+    "jss-theme-reactor": "^0.11.0",
     "lodash.camelcase": "^4.3.0",
     "mocha": "^3.2.0",
     "mocha-webpack": "^0.7.0",
-    "node-sass": "^4.5.0",
+    "murmurhash-js": "^1.0.0",
```

Comment:
lets move it to regular deps, so people dont have to install it on its own. im suspicious about it